### DISCUSSION TITLE
验证api出现未知错误，请求过于频繁无法对话

### DISCUSSION THREAD

**Discussion by Methaned** [GaiZhenbiao/ChuanhuChatGPT#511] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/129674003/229359604-118cc377-9689-4172-8ca4-c49af38f142e.png)
无论发什么都是请求过于频繁尝试验证api则会出现未知错误
![image](https://user-images.githubusercontent.com/129674003/229359662-6e8f64c0-10a9-47e2-b245-f896866f08e8.png)

**Comment by GaiZhenbiao** [comment, marked as answer]
大概率你的号被封了

**Comment by Handrail4015** [comment]
有同样的问题，而且检查api发现似乎没有被调用到

**Comment by Methaned** [comment]
确实是我的api有问题，账号倒是没有被封只封了api
